### DISCUSSION TITLE
Issue with OTP Delivery using egov-otp Service

### DISCUSSION THREAD

**Discussion by dhwani-ankit** [egovernments/DIGIT-DevOps#1818] [Q&A] [answered]
I am trying to send OTP using the egov-otp service, but I'm not receiving the OTP on my phone. Do we need to configure or add any credentials for this service, or is the egov-otp service sufficient to send OTPs?

**Comment by talele08** [comment, marked as answer]
You need to provide the credentials of your SMS provider in the secrets yaml file. Note: The default SMS provider implementation done is for SMS Country and you will need its account. If you wish to use some other SMS provider, you need to implement the third party provider API integration in egov-sms-notification

**Comment by m27wang** [comment]
I'm using **[otp.solutions]** to send OTP via Whatsapp, it's more easier, no delay time and no setup fee.
[https://otp.solutions](https://otp.solutions)
